src/Simplex/Messaging/Agent/Client.hs

```diff
@@ -2041,6 +2044,49 @@ getAgentWorkersSummary AgentClient {smpClients, ntfClients, xftpClients, smpDeli
             (pure WorkersSummary {numActive, numIdle = numIdle + 1, totalRestarts = totalRestarts + restartCount})
             (pure WorkersSummary {numActive = numActive + 1, numIdle, totalRestarts = totalRestarts + restartCount})
 
+data AgentQueuesInfo = AgentQueuesInfo
+  { msgQInfo :: TBQueueInfo,
+    subQInfo :: TBQueueInfo,
+    smpClientsQueues :: Map String (Either (AgentErrorType, Maybe UTCTime) ClientQueuesInfo)
```

Comment:
```suggestion
    smpClientsQueues :: Map String (Maybe (Either (AgentErrorType, Maybe UTCTime) ClientQueuesInfo))
```

Although it maybe better to use some other type, not Maybe, to see the clients that are in the process of connecting.

Reply:
otherwise they will be lost in JSON

Reply:
or we can use JSON option to not omit `null`s for AgentQueuesInfo (but I don't know how it'll pass via Map)